internal/web_server/web/views/components/error_dialog.templ

```diff
@@ -0,0 +1,58 @@
+package components
+
+const (
+	errorDialogId  = "error-dialog"
+	errorMessageId = "error-message"
+)
+
+// Error dialog with given title and message.
+//
+// If placed in a page, it will catch any htmx response with status code >= 400 and display the error dialog with error details.
+templ ErrorDialog(title string, message string) {
+	<dialog
+		id={ errorDialogId }
+		class="px-10 py-10 border-2 border-red-500 border-solid"
```

Comment:
If `px-10` and `py-10` is equal, you can do `p-10`, but it usually looks better if `px` is larger `py`.